approved/j750/prb1_flow.txt

```diff
@@ -119,22 +123,26 @@ DFF 1.1	Flow Table
 						Test	meas_ps_leakage	meas_ps_leakage	1170		97		6	Fail														
 						logprint	Speed_binning_example_bug_from_video_5																					
 						Test	test200_1							None		g200_7AF57D1_FAILED_0												
-						Test	test200_2							None		g200_7AF57D1_FAILED_1												
-						Test	test200_3							None		g200_7AF57D1_FAILED_2												
 						flag-true	g200_7AF57D1_FAILED																flag-true	g200_7AF57D1_FAILED_0				
+						Test	test200_2							None		g200_7AF57D1_FAILED_1												
 						flag-true	g200_7AF57D1_FAILED																flag-true	g200_7AF57D1_FAILED_1				
+						Test	test200_3							None		g200_7AF57D1_FAILED_2												
 						flag-true	g200_7AF57D1_FAILED																flag-true	g200_7AF57D1_FAILED_2				
-						Test	test100_1				5			Fail	g100_7AF57D1_RAN_0	g100_7AF57D1_RAN_0							flag-true	g200_7AF57D1_FAILED				
-						Test	test100_2				5			Fail	g100_7AF57D1_RAN_1	g100_7AF57D1_RAN_1							flag-true	g200_7AF57D1_FAILED				
-						Test	test100_3				5			Fail	g100_7AF57D1_RAN_2	g100_7AF57D1_RAN_2							flag-true	g200_7AF57D1_FAILED				
-						flag-true	g100_7AF57D1_RAN																flag-true	g100_7AF57D1_RAN_0				
-						flag-true	g100_7AF57D1_RAN																flag-true	g100_7AF57D1_RAN_1				
-						flag-true	g100_7AF57D1_RAN																flag-true	g100_7AF57D1_RAN_2				
+						flag-true	g100_7AF57D1_RAN																flag-true	g200_7AF57D1_FAILED				
+						Test	test100_1				5			Fail									flag-true	g200_7AF57D1_FAILED				
+						Test	test100_2				5			Fail									flag-true	g200_7AF57D1_FAILED				
+						Test	test100_3				5			Fail									flag-true	g200_7AF57D1_FAILED				
 						set-device				2				Pass									flag-true	g100_7AF57D1_RAN				
 						logprint	Test_node_optimization_within_an_if_failed_branch																					
 						Test	some_func_test	some_func_test						None		sft1_7AF57D1_FAILED												
-						set-device					10			Fail									flag-true	alarm				
-						set-device					11			Fail								not	flag-true	alarm				
+						flag-true	sft1_7AF57D1_FAILED_AND_Alarm																					
+						flag-false	sft1_7AF57D1_FAILED_AND_Alarm															not	flag-true	Alarm				
+						flag-false	sft1_7AF57D1_FAILED_AND_Alarm															not	flag-true	sft1_7AF57D1_FAILED				
+						set-device					10			Fail									flag-true	sft1_7AF57D1_FAILED_AND_Alarm				
+						flag-true	sft1_7AF57D1_FAILED_AND_NOT_Alarm																					
+						flag-false	sft1_7AF57D1_FAILED_AND_NOT_Alarm																flag-true	Alarm				
+						flag-false	sft1_7AF57D1_FAILED_AND_NOT_Alarm															not	flag-true	sft1_7AF57D1_FAILED				
+						set-device					11			Fail								not	flag-true	sft1_7AF57D1_FAILED_AND_NOT_Alarm				

```

Comment:
This is an improvement, the original generation of this sequence was actually completely wrong and did not gate the set-device by the result of some_func_test.
The new output is correct.